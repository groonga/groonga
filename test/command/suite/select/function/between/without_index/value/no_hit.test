table_create Users TABLE_HASH_KEY ShortText
column_create Users age COLUMN_SCALAR Int32

load --table Users
[
{"_key": "alice",  "age": 17},
{"_key": "bob",    "age": 18},
{"_key": "calros", "age": 19},
{"_key": "dave",   "age": 20},
{"_key": "eric",   "age": 21}
]

select Users --filter 'between(17, 18, "include", 20, "exclude")'
