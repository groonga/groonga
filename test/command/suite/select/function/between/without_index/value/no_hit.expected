table_create Users TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Users age COLUMN_SCALAR Int32
[[0,0.0,0.0],true]
load --table Users
[
{"_key": "alice",  "age": 17},
{"_key": "bob",    "age": 18},
{"_key": "calros", "age": 19},
{"_key": "dave",   "age": 20},
{"_key": "eric",   "age": 21}
]
[[0,0.0,0.0],5]
select Users --filter 'between(17, 18, "include", 20, "exclude")'
[[0,0.0,0.0],[[[0],[["_id","UInt32"],["_key","ShortText"],["age","Int32"]]]]]
